content/src/content/jcr_root/apps/core/wcm/components/contentfragment/v1/contentfragment/templates.html

```diff
@@ -43,6 +43,9 @@ <h3 class="cmp-contentfragment__title">${fragment.title}</h3>
             ${item @ context="html"}
             <div data-sly-resource="${'par{0}' @ format=itemList.count, resourceType=fragment.gridResourceType}"></div>
         </div>
+        <div data-sly-list="${fragment.remainingInBetweenContent}">
```

Comment:
I think this addition should be behind an option so that we don't end up appending these where they would not be desired. IMO, it's best to have a content policy option to enable showing them.